sympy/polys/matrices/domainmatrix.py

```diff
@@ -57,13 +98,78 @@ def get_domain(cls, items_sympy, **kwargs):
         return K, items_K
 
     def convert_to(self, K):
+        r"""
+            Returns a DomainMatrix with the desired domain or field
+
+            Usage
+            =====
+
+            >>> from sympy import Matrix, CC
+            >>> from sympy.polys.matrices import DomainMatrix
+            >>> Matrix1 = Matrix([
+            ...         [2, 73],
+            ...         [27, 86]])
+            >>> domainMatrix = DomainMatrix.from_Matrix(Matrix1)
+            >>> domainMatrix
+            DomainMatrix([[2, 73], [27, 86]], (2, 2), ZZ)
+
+            >>> domainMatrix.convert_to(CC)
+            DomainMatrix([[(2.0 + 0.0j), (73.0 + 0.0j)],
+            [(27.0 + 0.0j), (86.0 + 0.0j)]], (2, 2), CC)
+
+        """
         return self.from_rep(self.rep.convert_to(K))
 
     def to_field(self):
+        r"""
+            Returns a DomainMatrix with the appropriate field
+
+            Usage
+            =====
+
+            >>> from sympy import Matrix
+            >>> from sympy.polys.matrices import DomainMatrix
+            >>> Matrix1 = Matrix([
+            ...         [2, 73],
+            ...         [27, 86]])
+            >>> domainMatrix = DomainMatrix.from_Matrix(Matrix1)
+            >>> domainMatrix
+            DomainMatrix([[2, 73], [27, 86]], (2, 2), ZZ)
+
+            >>> domainMatrix.to_field()
+            DomainMatrix([[2, 73], [27, 86]], (2, 2), QQ)
+
+        """
         K = self.domain.get_field()
         return self.convert_to(K)
 
     def unify(self, other):
+        r"""
+            Unifies any two DomainMatrix with different domains
+
+            Usage
+            =====
+
+            >>> from sympy import Matrix
+            >>> from sympy.polys.matrices import DomainMatrix
+            >>> Matrix1 = Matrix([
+            ...         [2, 73],
+            ...         [27, 86]])
+            >>> domainMatrix = DomainMatrix.from_Matrix(Matrix1)
+            >>> Matrix2 = Matrix([
+            ...         [19, 30],
+            ...         [56, 14]])
+            >>> domainMatrix1 = DomainMatrix.from_Matrix(Matrix2)
+            >>> domainMatrix
+            DomainMatrix([[2, 73], [27, 86]], (2, 2), ZZ)
+            >>> domainMatrix1
+            DomainMatrix([[19, 30], [56, 14]], (2, 2), ZZ)
+
+            >>> domainMatrix.unify(domainMatrix1)
+            (DomainMatrix([[2, 73], [27, 86]], (2, 2), ZZ),
+            DomainMatrix([[19, 30], [56, 14]], (2, 2), ZZ))
```

Comment:
The example would be much more interesting if the two matrices had different domains.